### PR TITLE
[BPK-1951] Add performance story for Label

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

File: Example/Backpack/View Controllers/LabelsPerformanceViewController.swift [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        }
+    }
+
+    static func buildHorizontalStackView(emphasized: Bool) -> UIStackView {
+        let horizontalStackView = UIStackView(frame: CGRect.zero)
+        horizontalStackView.axis = .horizontal
+        horizontalStackView.spacing = BPKSpacingSm
+        horizontalStackView.distribution = .equalSpacing
+
+        let l1 = Backpack.Label(fontStyle: emphasized ? BPKFontStyle.textXs : BPKFontStyle.textXsEmphasized)
```
Comment (georgegillams) [inline Example/Backpack/View Controllers/LabelsPerformanceViewController.swift]: Could we iterate over a list of text-styles instead to generate these without so much duplication?

Comment (k0nserv) [reply Example/Backpack/View Controllers/LabelsPerformanceViewController.swift]: Yeah sure

Comment (k0nserv) [reply Example/Backpack/View Controllers/LabelsPerformanceViewController.swift]: Done

**Comment by georgegillams** [comment]
That was rapid!

fast coding

**Pull request merged**